packages/manager/modules/pci/src/projects/project/ssh-keys/ssh-keys.html

```diff
@@ -25,17 +25,18 @@
 
 <oui-datagrid data-rows="$ctrl.keys">
     <oui-datagrid-topbar>
-        <a
+        <button
```

Comment:
No `oui-button` here ?
If it's because of the tracking, could you handle the tracking in the routing file, so we can have a consistent usage of oui-button please